src/cdk-experimental/menu/menu.md

```diff
@@ -0,0 +1,240 @@
+items within an inline menus are logically grouped together.
+
+<!-- TODO inline menu example (gmail buttons?) -->
+
+### Menu Items
+
+Components which interact with or are placed inside of a menu or menubar should typically have the
+`cdkMenuItem` directive provided. This directive allows the items to be navigated to via keyboard
+interaction.
+
+A menuitem by itself can provide some user defined action by hooking into the `cdkMenuItemTriggered`
+output. An example may be a close button which performs some closing logic.
+
+```html
+<ng-template cdkMenuPanel #panel="cdkMenuPanel">
+  <div cdkMenu [cdkMenuPanel]="panel">
+    <button cdkMenuItem [cdkMenuItemTriggered]="closeApp()">Close</button>
+  </div>
+</ng-template>
+```
+
+`cdkMenuItem` should also be added to a component which triggers a menu or submenu.
+
+```html
+<ng-template cdkMenuPanel #panel="cdkMenuPanel">
+  <div cdkMenu [cdkMenuPanel]="panel">
+    <button cdkMenuItem [cdkMenuTriggerFor]="submenu">Open Submenu</button>
+  </div>
+</ng-template>
+```
+
+A menuitem also has two sub-types, neither of which should trigger a menu: CdkMenuItemCheckbox and
+CdkMenuItemRadio
+
+#### Menu Item Checkboxes
+
+A component with the `cdkMenuItemCheckbox` directive behaves just as you would expect a typical
+checkbox to behave. A use case may be toggling an independent setting since it contains a checked
+state. A component with the `cdkMenuItemCheckbox` directive does not need the additional
+`cdkMenuItem` directive.
+
+#### Menu Item Radios
+
+A `cdkMenuItemRadio` behaves as you would expect a radio button to behave. A use case may be
+toggling through more than one state since radio buttons can be grouped together. A component with
+the `cdkMenuItemRadio` directive does not need the additional `cdkMenuItem` directive.
+
+#### Groups
+
+By default `cdkMenu` acts as a group for `cdkMenuItemRadio` elements. That is, components marked
+with `cdkMenuItemRadio` added as children of a `cdkMenu` will be logically grouped and only a single
+component can have the checked state.
+
+If you would like to have unrelated groups of radio buttons within a single menu you should use the
+`cdkMenuGroup` directive.
+
+```html
+<ng-template cdkMenuPanel #panel="cdkMenuPanel">
+  <div cdkMenu [cdkMenuPanel]="panel">
+    <!-- Font size -->
+    <div cdkMenuGroup>
+      <button cdkMenuItemRadio>Small</button>
+      <button cdkMenuItemRadio>Medium</button>
+      <button cdkMenuItemRadio>Large</button>
+    </div>
+    <hr />
+    <!-- Paragraph alignment -->
+    <div cdkMenuGroup>
+      <button cdkMenuItemRadio>Left</button>
+      <button cdkMenuItemRadio>Center</button>
+      <button cdkMenuItemRadio>Right</button>
+    </div>
+  </div>
+</ng-template>
+```
+
+Note however that when the menu is closed and reopened any state is lost. You must subscribe to the
+groups `change` output, or to `cdkMenuItemToggled` on each radio item and track changes your self.
```

Comment:
How do you specify which item should be checked to start?

Reply:
There's a `checked` input - I'll add that to the doc.